api/src/main/java/com/codeforcommunity/api/IProcessor.java

```diff
@@ -1,21 +1,47 @@
 package com.codeforcommunity.api;
 
-import com.codeforcommunity.dto.MemberReturn;
+import com.codeforcommunity.dto.UserReturn;
+import com.codeforcommunity.dto.EventReturn;
 
 import java.util.List;
 import java.time.LocalDateTime;
 
 public interface IProcessor {
   /**
-   * Get all the members first and last names.
+   * Get all the users first and last names.
    */
-  List<MemberReturn> getAllMembers();
 
-  boolean attendMeeting(String meetingid, String username);
-  boolean createMeeting(String id, String name, LocalDateTime date, boolean open);
-  boolean addMember(String first, String last);
-  boolean validate(String first, String last);
+  List<UserReturn> getEventUsers(int eventId);
+
+  boolean attendEvent(String eventCode, int userId);
+
+  List<EventReturn> getAllEvents();
+
+  boolean createEvent(String name, LocalDateTime date, boolean open, String code);
```

Comment:
Try to be consistent with parameter names, if you use `eventCode` above use that here too instead of `code`